src/utilities/util_verbose.c

```diff
@@ -47,7 +47,7 @@ __handle_progress_verbose(
     WT_UNUSED(handler);
     WT_UNUSED(session);
 
-    return (printf("\r\t%s %-20" PRIu64, operation, progress) < 0 ? EIO : 0);
+    return (printf("\r\t%s %-20" PRIu64 "\n", operation, progress) < 0 ? EIO : 0);
```

Comment:
I made this change since after I moved the new `wt verify` output under the `-v` flag it started to blend with this output in text. 

I personally found that adding new line here looks better, and I didn't find any other places in CI where we do use it. But if you know any customers/tests that can be affected or just think that this shouldn't be changed for some reason - please let me know.

Reply:
Do you have a before / after output?